API/NoDocs/Microsoft.AspNetCore.Diagnostics.Latency/RequestCheckpointConstants.cs

```diff
@@ -0,0 +1,12 @@
+// Assembly 'Microsoft.AspNetCore.Diagnostics.Middleware'
+
+namespace Microsoft.AspNetCore.Diagnostics.Latency;
+
+public static class RequestCheckpointConstants
+{
+    public const string ElapsedTillHeaders = "elthdr";
+    public const string ElapsedTillFinished = "eltltf";
+    public const string ElapsedTillPipelineExitMiddleware = "eltexm";
+    public const string ElapsedResponseProcessed = "eltrspproc";
+    public const string ElapsedTillEntryMiddleware = "eltenm";
+}
```

Comment:
Are these standard? Is there prior art? Are these header names?

Reply:
These are not headers, not standard. They are emitted by the latency infra in a log record to hold checkpoints.